catalogue/webapp/test/utils/dates.test.ts

```diff
@@ -128,23 +128,23 @@ describe('findClosedDays', () => {
   });
 });
 
-describe('findNextRegularOpenDay: finds the earliest date on which the venue is normally open', () => {
+describe('findNextPickUpDay: finds the earliest date on which requested items can be picked up', () => {
   it('returns the same date provided if it occurs on one of the regular open days', () => {
-    const result = findNextRegularOpenDay(
+    const result = findNextPickUpDay(
       london('2022-01-15'), // Saturday
       [0, 1, 2] // Sunday, Monday, Tuesday
     );
     expect(result.toDate()).toEqual(london('2022-01-15').toDate()); // Saturday
   });
```

Comment:
I've convinced myself that this is correct, but the lack of named variables made it decidedly non-obvious. Since you're improving the name of this function, can we also improve the readability? Something like:

```typescript
The lack of named variables in the test make this somewhat difficult to follow; I think something like:

```typescript
findNextPickUpDay({
  onOrAfter: London('2022-01-15'), // Saturday
  regularClosedDays: [0, 1, 2], // Sunday, Monday, Tuesday
})
```

because this function isn't being called with _today's_ date (in which case pickup on the Saturday is wrong), it's being called with the first date we think is suitable.